tests/wpt/mozilla/meta/MANIFEST.json

```diff
@@ -13775,7 +13775,7 @@
      ]
     ],
     "interfaces.https.html": [
-     "1397b723a1cb001521ac1b2032c380f1e02cf1f0",
+     "027e138f4af5e2a13df9bf8c78765dcf57f6b611",
```

Comment:
Requires resolution and merging in https://github.com/whatwg/fetch/pull/1859 first

Reply:
Spec was updated to make it only available on a secure context